exercises/concept/fruit-picker/.docs/instructions.md

```diff
@@ -2,21 +2,27 @@
 
 You are creating a new online portal for your patrons to order their fruit fresh from the grocer. The grocer has an API that you can use to see if they have the inventory desired by your customers. You need to create a small library of functions for interacting with the grocer's API.
 
-## 1. Check if the grocer's service is online
+## 1. Create a callback to be called when the order is successful
 
-The grocer's application programming interface [API] provides a function to check if their service is online called `checkStatus`. Use the grocer's function to finish the implementation of your `isServiceOnline` function. The `checkStatus` function takes a callback function to receive the status from the API.
+Write a callback function called `onSuccess` to be called when the order is successful. It should invoke the imported `notify` function passing a success message to it.
 
-- if the status is `'ONLINE'`, return `true`
-- if the status is `'OFFLINE'`, return `false`
+```javascript
+onSuccess();
+// => `notify` called with `{ message: 'SUCCESS' }`
+```
+
+## 2. Create a callback to be called when the order fails with an error
```

Comment:
Its a bit sad that task 1 and 2 now don't have any story relation anymore. Also the headers are very much focused on the programming part now. Not a blocker for merging, just an observation in case you have an idea how to improve.

Reply:
I don't have an idea how to improve it, but if there is a discrete action/improvement you can suggest I can implement